cypress/e2e/ui/Settings/Application-Settings/edit_collect_logs.cy.js

```diff
@@ -0,0 +1,288 @@
+/* eslint-disable no-undef */
+
+function disableNotificationsIfVisible() {
+  // Look for notification popups and disable them if present
+  cy.get('body').then(($body) => {
+    const $link = $body.find(
+      '.miq-toast-wrapper .row .alert a:contains("Disable notifications")'
+    );
+    if ($link.length && $link.is(':visible')) {
+      cy.wrap($link).click({ force: true });
+    }
+  });
+}
+
+function resetProtocolForServer() {
+  // Select Diagnostics
+  cy.get('.panel #control_diagnostics_accord .panel-title a')
+    .contains('Diagnostics')
+    .click();
+  // Open ManageIQ Region: - list view if not already open
+  cy.get(
+    '#diagnostics_accord .treeview li.list-group-item[title^="ManageIQ Region:"]'
+  ).then(($li) => {
+    const span = $li.find('span.fa-angle-right');
+    if (span.length) {
+      cy.wrap(span).click();
+    }
+  });
+  // Open Zone: - list view if not already open
+  cy.get(
+    '#diagnostics_accord .treeview li.list-group-item[title^="Zone:"]'
+  ).then(($li) => {
+    const span = $li.find('span.fa-angle-right');
+    if (span.length) {
+      cy.wrap(span).click();
+    }
+  });
+  // Selecting Server: list view
+  cy.get(
+    '#diagnostics_accord .treeview li.list-group-item[title^="Server:"]'
+  ).click();
+  // Selecting Collect Logs nav bar
+  cy.get(
+    '#tab_all_tabs_div #ops_tabs .nav-tabs li#diagnostics_collect_logs_tab'
+  ).click();
+  // Clicking Edit button
+  cy.get(
+    '.miq-toolbar-actions .miq-toolbar-group button#log_depot_edit'
+  ).click();
+  cy.wait('@editEventForServer');
+  // Resetting Protocol dropdown value
+  cy.get('#log-depot-settings .bx--select select#log_protocol').then(
+    ($select) => {
+      const currentValue = $select.val();
+      // If the value is not default one(BLANK_VALUE), then setting it to blank
+      if (currentValue !== 'BLANK_VALUE') {
+        cy.wrap($select).select('BLANK_VALUE');
+        cy.get('#diagnostics_collect_logs .bx--btn-set button[type="Submit"]')
+          .contains('Save')
+          .click();
+        cy.get('#main_div #flash_msg_div .alert-success').contains(
+          'Log Depot Settings were saved'
```

Comment:
When https://github.com/ManageIQ/manageiq-ui-classic/pull/9504 gets merged, we can use the helper function here.

Reply:
Note, checking for "saved" in the message is probably all we need.